### ISSUE TITLE
Can "--report" show a summary much more simple?

### ISSUE THREAD

**Issue by warcayac** [athityakumar/colorls#381]
### Description

- Relevant Issues : 
Using "--report" parameter shows a summary of several lines, like this:
```
Found 14 contents in directory /home/user/WDisk02/Projects/Flutter/audioplayers_example

        Folders                 : 7
        Recognized files        : 7
        Unrecognized files      : 0
```
There is much space being wasted (IMHO), it would be more fancy if it showed something only like this:
```
        Folders: 7 , Files: 7
```

- Relevant PRs : (none)
- Type of issue :
  - [ ] Installation
  - [ ] Font-related
  - [X] **Feature request**
  - [ ] Bug in existing feature
  - [ ] Developer mode : Code quality / Tests / Documentation

**Comment by avdv** [comment]
I think this could be done by making the `--report` option taking a parameter, e.g. `--report=short`.

**Comment by goar5670** [comment]
Hello, I want to contribute to the repo, can I get this issue assigned to me?

**Comment by avdv** [comment]
Hi @goar5670 :wave: 

nice that you want to help out!

**Comment by avdv** [comment]
Fixed by #502
